server/data/data-jmap/src/main/java/org/apache/james/jmap/api/preview/Preview.java

```diff
@@ -31,17 +31,14 @@
 public class Preview {
 
     public static final int MAX_LENGTH = 256;
-    public static final Preview NO_BODY = Preview.from("(Empty)");
+    public static final Preview NO_BODY = new Preview("(Empty)");
 
-    public static Preview from(String value) {
```

Comment:
does that mean when reading the preview from the database, it will go into the transformation pipeline once again?

Reply:
oh, you're right!